src/puppetlabs/puppetdb/jdbc.clj

```diff
@@ -288,7 +290,7 @@
      (fn []
        ~@body)))
 
-(defn with-query-results-cursor*
+(defn with-query-results-cursor
```

Comment:
I think we should call this streamed-query-result now; it makes more sense at the call site.